### PR TITLE
formatting update for RFQ

### PR DESCRIPTION
**Pull request by jehlers**
put "(vii) Date of submission" into the block quote, matched "a." formatting with their attached bolded or italicized headers, adding links to attached documents

### CODE REVIEW

File: RFQ.md [hunk 1]
```diff
@@ -282,15 +282,15 @@ Schedule and Agenda for Oral Presentations:
...
 
-#### b. *Part One.* Part One of the offeror’s oral presentation will consist of the offeror answering the Government’s core questions related to the technical factors identified in the Request For Quotation (RFQ). While the technical factors are identified in the RFQ, the core questions are not. All offerors will be asked the same core questions related to the technical factors. The questions are related to the offeror’s technical understanding and approach of the requirements identified in the Performance Work Statement (PWS) and quality control measures for overseeing performance. This part of the oral presentation of answers will not exceed 25 minutes, and may go shorter.
```
Comment (jehlers) [inline RFQ.md]: @billy22g or @allalala or @stvnrlly or @estherkim18f or whoever ... Is there an "a" to this "b"?

File: RFQ.md [hunk 2]
```diff
@@ -282,15 +282,15 @@ Schedule and Agenda for Oral Presentations:
...
+#### *b.* *Part One.* Part One of the offeror’s oral presentation will consist of the offeror answering the Government’s core questions related to the technical factors identified in the Request For Quotation (RFQ). While the technical factors are identified in the RFQ, the core questions are not. All offerors will be asked the same core questions related to the technical factors. The questions are related to the offeror’s technical understanding and approach of the requirements identified in the Performance Work Statement (PWS) and quality control measures for overseeing performance. This part of the oral presentation of answers will not exceed 25 minutes, and may go shorter.
```
Comment (jehlers) [inline RFQ.md]: @billy22g or @allalala or @stvnrlly or @estherkim18f or anyone else... is there an "a" to this "b?"

Comment (stvnrlly) [inline RFQ.md]: Hmm, there probably _should have been_ an "a", but if that's in what's posted we should keep it as is unless it's changed there, as well.

**Comment by billy22g** [comment]
There were already changes that conflicted with this @jehlers, so I couldn't merge easily. I think I hit everything you did, but let me know if you see anything that's not there in the master branch. If you're 👍 I'll close this PR.

**Comment by stvnrlly** [comment]
No worries; there were no conflicts when it was submitted, but there were definitely a lot of commits flying around so something was bound to happen. I'm happy to resolve the conflicts if that'd be useful.

**Comment by billy22g** [comment]
Sorry, I didn't mean to suggest that you created the conflicts! @stvnrlly and I had been working on the same doc and I pushed to master, which created the conflicts. I think I hit all the things you did on the current version, so should be all good now. Thanks again @jehlers!

**Pull request closed without merging**